### PR TITLE
Add release notes for 1.12.3

### PR DESCRIPTION
**Pull request by morphis**
Still WIP as we miss updated component versions

### CODE REVIEW

File: announcements/1.12.3.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* AC-649 Don't return an error if JoinSession is called without a body
+* AC-648 Dashboard ends up with no certificates
+* AC-644 LP#1955986 Application version got stuck to "initializing" when changing an application attribute via `amc application set`
+* AC-627 GPU ends up with zero slots when added after initial node creation
+* AC-625 nvidia\_drm kernel module isn't loaded after appliance upgrade
+* AC-277 LP #1922889: The `system.log` and `android.log` are missing and not collected by AMS when the container ran into an error
+
+#### Upgrade instructions
+
+See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) or [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.12.2 release.
```
Comment (ru-fu) [inline announcements/1.12.3.md:23]:
```suggestion
See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) or [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.12.3 release.
```

File: announcements/1.12.3.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.12.3.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+ * Android security updates for January 2021 (see [Android Security Bulletin - January 2021](https://source.android.com/security/bulletin/2022-01-01) for more information)
```
Comment (ru-fu) [inline announcements/1.12.3.md:9]:
```suggestion
 * Android security updates for January 2022 (see [Android Security Bulletin - January 2022](https://source.android.com/security/bulletin/2022-01-01) for more information)
```

File: release-notes.md [hunk 3]
```diff
@@ -2,6 +2,26 @@
 
 See [Upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [Upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for instructions on how to update your Anbox Cloud deployment.
 
+[Details=1.12.3]
+
+## 1.12.3 (January 20 2022)
+
+### New features & improvements
+
+ * Android security updates for January 2021 (see [Android Security Bulletin - January 2021](https://source.android.com/security/bulletin/2022-01-01) for more information)
```
Comment (ru-fu) [inline release-notes.md:11]:
```suggestion
 * Android security updates for January 2022 (see [Android Security Bulletin - January 2022](https://source.android.com/security/bulletin/2022-01-01) for more information)
```

**Review by ru-fu — CHANGES_REQUESTED** [review]
Welcome to the future. ;)

**Review by morphis — COMMENTED** [review]
Thanks, not sure how I missed that we have 2022 ... :-)

**Review by ru-fu — APPROVED** [review]
Looks good for now!

**Pull request merged**